misc/connection-limits/CHANGELOG.md

```diff
@@ -1,9 +1,12 @@
 ## 0.5.1
 
-- Allow setting Peer IDs for bypassing limit check. 
+- Allow setting Peer IDs for bypassing limit check.
   Connections to the specified peers won't be counted toward limits.
   See [PR 5720](https://github.com/libp2p/rust-libp2p/pull/5720).
 
+- Added test for newly exposed `Option<PeerId>` in `SwarmEvent::IncomingConnectionError`
+  See [PR 6032](https://github.com/libp2p/rust-libp2p/pull/6032).
+
```

Comment:
```suggestion
```

No changelog entry needed for tests.

Reply:
I only added this because CI gave me an error, saying that since I changed the file a log entry must also be made :shrug: We'll see if it passes without it this time